File: sol/solution/8 kyu/5a0be7ea8ba914fc9c00006b/Solution.py
#
# Solution
#
def sakura_fall(v):
    if v <= 0:
        return 0.0
    return 400.0 / v





#
# Sample Tests
#
import codewars_test as test
from solution import sakura_fall

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_approx_equals(sakura_fall(5), 80)
        test.assert_approx_equals(sakura_fall(10), 40)
        test.assert_approx_equals(sakura_fall(-1), 0)

File: sol/solution/8 kyu/5a0be7ea8ba914fc9c00006b/Solution.rb
=begin
Solution
=end
def sakura_fall(v)
  return 0.0 if v <= 0
  400.0 / v
end





=begin
Sample Tests
=end
describe "Solution" do
    $DELTA = 1e-6

  it "Fixed tests" do
    expect(sakura_fall(5)).to be_within($DELTA).of(80)
    expect(sakura_fall(10)).to be_within($DELTA).of(40)
    expect(sakura_fall(200)).to be_within($DELTA).of(2)
    expect(sakura_fall(-1)).to be_within($DELTA).of(0)
  end
end